apphub/image_generation/pggan/pggan_tf.py

```diff
@@ -310,6 +311,7 @@ def forward(self, data, state):
         return tf.reduce_mean(loss)
 
 
+@traceable()  # to make this trace restorable
```

Comment:
Do we want to do this in our apphub?

Reply:
I'm thinking probably not since the app hub doesn't use the restore wizard anyways. if we do want it then we would also need to put it in the torch file

Reply:
I thought it before but it is weird enough that pytorch script didn't throw an error as tf does if i didn't put traceable decorator there